docs/eta-pi-p/manual.ipynb

```diff
@@ -383,6 +615,71 @@
     "fig.tight_layout()\n",
     "plt.show(fig)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Phase space"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phsp = helicity_transformer(phsp_momenta)\n",
```

Comment:
I would move the definition of the `helicity_transformer` and of `phsp_momenta` closer together. They all have to do with 'data' (numerical preparation for the plots), which is the previous section.